atomic_reactor/util.py

```diff
@@ -652,6 +650,16 @@ def get_credentials(self, docker_registry):
             logger.warn('%s not found in .dockercfg', docker_registry)
             return {}
 
+    def _log_and_raise_error(self, msg):
+        """ Helper method that raise and log a
+        RuntimError exception
+
+        :param msg: message displayed in the log
+                    and in the exception
+        """
+        logger.error(msg, exc_info=True)
```

Comment:
The place where you factored this out of was specifically handling catching an exception and rethrowing it in simplified form (while logging the original exception) - it doesn't make sense to me as a general way of raising an error. You can just raise RuntimeError("dockercfg secret file does not exist")

Reply:
Good point